code/modules/admin/verbs/massmodvar.dm

```diff
@@ -441,5 +445,48 @@
 						if (A.type == O.type)
 							A.vars[variable] = O.vars[variable]
 
+		if ("matrix")
+			var/matrix/new_value = modify_matrix_menu(O.vars[variable], verbose = FALSE)
+			if (!new_value)
+				return
+
+			O.vars[variable] = new_value
```

Comment:
Tell me this block isn't copypasted, over and over and over.
Tell me it ain't so.